index.bs

```diff
@@ -1451,11 +1448,11 @@ dictionary MessageArgs {
 
 
 ### SIMID:Creative:fatalError ### {#simid-creative-fatalError}
-The creative posts `SIMID:Creative:fatalError` in cases when its internal exceptions prevent the interactive component from further execution. In response to the `fatalError` message, the player unloads the SIMID iframe and, if possible, ends ad media playback. 
+The creative posts `SIMID:Creative:fatalError` in cases when its internal exceptions prevent the interactive component from further execution. In response to the `Creative:fatalError` message, the player unloads the SIMID iframe and reports VAST error tracker with the `errorCode` specified by the creative. The ad media playback continues.
```

Comment:
I don't think we should change the media playback behavior.

IMO the sentence:
The ad media playback continues.  

Should be:
If possible, the ad media playback is stopped.  This may not be possible with SSAI.